superset-frontend/src/views/CRUD/welcome/EmptyState.tsx

```diff
@@ -151,10 +151,9 @@ export default function EmptyState({ tableName, tab }: EmptyStateProps) {
             window.location.href = favRedirects[tableName];
           }}
         >
-          See all{' '}
-          {tableName === 'SAVED_QUERIES'
+          { t('See all %s', tableName === 'SAVED_QUERIES'
             ? t('SQL Lab queries')
-            : t(`${tableName}`)}
+            : t(`${tableName}`))}
```

Comment:
Could we rather split this up to be explicit, like 
```ts
tableName === 'SAVED_QUERIES' ? t('See all saved queries') : welcomeTableLabels[tableName]`
```
Here we can use `welcomeTableLabels[tableName]` directly, as it already has the translated strings as the values.

Reply:
You mean like this:

```
{t('See all %(tableName)s', {
  tableName: welcomeTableLabels[tableName],
})}
```
?

Reply:
done